### PR TITLE
Add config to allow defining a default tax code for shipping services

### PR DESCRIPTION
**Pull request by mttdffy**
WORKAREA-256

### CODE REVIEW

File: core/lib/workarea/configuration.rb [hunk 1]
```diff
@@ -1306,6 +1306,12 @@ def self.setup_defaults
         cvv
         amount
       ]
+
+      # Tax code assigned to shipping options when an existing service does
+      # not exist. This is useful for third-party gateways to assign tax codes
+      # to dynamically generated options. This can either be a string or a proc.
+      # If using a proc, the carrier and name of the service will be passed.
+      config.default_shipping_service_tax_code = nil
```
Comment (bencrouse) [inline core/lib/workarea/configuration.rb:1314]: Any reason we shouldn't make this admin facing?

Comment (mttdffy) [reply core/lib/workarea/configuration.rb:1314]: admin can't define procs?

Comment (bencrouse) [reply core/lib/workarea/configuration.rb:1314]: the code looks like it would support both!

Comment (mttdffy) [reply core/lib/workarea/configuration.rb:1314]: Not sure what you're saying

**Review by bencrouse — APPROVED** [review]

**Pull request merged**